readme.md

```diff
@@ -358,7 +358,7 @@
 - [Construct 2](https://github.com/WebCreationClub/awesome-construct#readme) - Game engine.
 - [Gideros](https://github.com/stetso/awesome-gideros#readme) - Game engine.
 - [Minecraft](https://github.com/bs-community/awesome-minecraft#readme) - Sandbox video game.
-
+- [Game Datasets](https://github.com/leomaurodesenv/game-datasets) - Game datasets and tools for Artificial Intelligence.
```

Comment:
The description should succinctly explain what game datasets are.

Reply:
You're also missing the `#readme`.

Reply:
> The description should succinctly explain what game datasets are.

What do you think "Tools for Artificial Intelligence in Games."?